### PR TITLE
Fix: shared view gallery 

### LINKED ISSUE #6439: 🐛 Bug: Shared gallery view
### Please confirm if bug report does NOT exists already ?

- [X] I confirm there is no existing issue for this

### Steps to reproduce ?

- [x] Shared view : click on card should open expanded record
- [x] Padding at top of text should be same as padding at bottom
Screenshot 2023-09-22 at 3 38 01 PM
- [x] Shared view, Edit cards : cover image field should be Read Only
- [x] Configure only cover image in gallery view - hide all fields. Shared view doesn't show up as expected
Screenshot 2023-09-22 at 3 43 22 PM
- [x] Hiding field didn't remove information from the shred view
Screenshot 2023-09-22 at 3 50 46 PM

### Desired Behavior

x

### Project Details

x

### Attachments

_No response_

### PR DESCRIPTION
**Pull request by DarkPhoenix2704**
## Change Summary

Provide summary of changes with issue number if any.

Closes #6439

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [x] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Test/ Verification

Provide summary of changes.

## Additional information / screenshots (optional)

Anything for maintainers to be made aware of

### CODE REVIEW

File: packages/nc-gui/composables/useExpandedFormStore.ts [hunk 1]
```diff
@@ -181,6 +179,7 @@ const [useProvideExpandedFormStore, useExpandedFormStore] = useInjectionState((m
               fn: async (rowData: any) => {
                 await $api.dbTableRow.create('noco', project.value.id as string, meta.value.id, { ...pkData, ...rowData })
                 if (activeView.value?.type === ViewTypes.KANBAN) {
+                  const { loadKanbanData } = useKanbanViewStoreOrThrow()
```
Comment (wingkwong) [inline packages/nc-gui/composables/useExpandedFormStore.ts:182]: same logic is duplicated 4 times. extract and make it as a function stead.

```ts
if (activeView.value?.type === ViewTypes.KANBAN) {
  const { loadKanbanData } = useKanbanViewStoreOrThrow()
  await loadKanbanData()
}
```

**Comment by dstala** [comment]
- [ ] If display value column is disabled, need some padding before card data display
- [ ] disable attachment upload option in shared gallery expand record

File: packages/nc-gui/composables/useExpandedFormStore.ts [hunk 2]
```diff
@@ -233,19 +235,16 @@ const [useProvideExpandedFormStore, useExpandedFormStore] = useInjectionState((m
...
                 },
                 args: [id, clone(updateOrInsertObj)],
               },
               undo: {
                 fn: async (id: string, data: Record<string, any>) => {
                   await $api.dbTableRow.update(NOCO, project.value.id as string, meta.value.id, encodeURIComponent(id), data)
-                  if (activeView.value?.type === ViewTypes.KANBAN) {
-                    await loadKanbanData()
-                  }
+                  loadKanbanData()
```
Comment (wingkwong) [inline packages/nc-gui/composables/useExpandedFormStore.ts:247]: await

**Review by dstala — APPROVED** [review]

File: packages/nc-gui/components/smartsheet/Gallery.vue [hunk 3]
```diff
@@ -319,7 +314,7 @@ watch(
               </h2>
 
               <div v-for="col in fieldsWithoutDisplay" :key="`record-${record.row.id}-${col.id}`">
-                <div class="flex flex-col ml-2 !pr-3.5 !mb-[0.75rem] rounded-lg w-full">
+                <div class="flex flex-col first:mt-3 ml-2 !pr-3.5 !mb-[0.75rem] rounded-lg w-full">
```
Comment (pranavxc) [inline packages/nc-gui/components/smartsheet/Gallery.vue:317]: What does `first:` means ?

**Pull request merged**